dogs.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="keywords" content="HTML, CSS, JavaScript">
+    <meta name="author"content="Elena Nam">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Are+You+Serious&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+    
+
+    <title>Dogs</title>
+</head>
+
+<body>
+    <main> 
+        <header>
+            <div class ="navbar">
+                 <nav id="nav">                          
+                 <a href="index.html"><strong> Home </strong></a>
+                 <a href="drinks.html"><strong> Coffee </strong></a> 
+                 <a class="active"><strong> Dogs </strong></a>
+                 <a href="reservation.html"><strong> Reservation </strong></a>
+                 </nav>
+            </div>
+        </header>
+
+        <div id = "dogs_page" >
+        <section id ="dogs">
+                <h2> Choose a dog </h2>
+
+            <div class="dog_card">
+            <div class="dog_row" id="dogs-list"></div>
+            <ul id="dogsName"></ul>
+            </div>
+            <div id="popup" class="popup" style="display: none;">
+                <div class="popup-content">
+                  <span class="close" onclick="closePopup()">&times;</span>
+                  <h3 id="popup-title">Dog Information</h3>
+                  <p><strong>Age:</strong> <span id="popup-age"></span></p>
+                  <p><strong>Sex:</strong> <span id="popup-sex"></span></p>
+                  <p><strong>Size:</strong> <span id="popup-size"></span></p>
+                  <p><strong>Location:</strong> <span id="popup-location"></span></p>
+                </div>
+              </div>
+                </section>
+        
+        </div> <!-- closing div for container-main -->
+
+        <div class = "container_two_buttons">
+            <button class="back"><a href="index.html">&laquo; Back </a></button>
+            <button class="next"><a href="reservation.html">Next &raquo;</a></button>
+            </div>
+            <footer>
+                <a href = "https://www.adoptapet.com" target="_blank"><img src = "adoptapet.png" alt = "logo of adoptapet" width= 200px;></a>
+            </footer>
```

Comment:
I like how this footer acts as a placeholder until the information about the dogs has loaded.